src/components/structures/FilePanel.tsx

```diff
@@ -206,7 +214,7 @@ class FilePanel extends React.Component<IProps, IState> {
                 // event index to populate the timelineSet for us. This call
                 // will add 10 events to the live timeline of the set. More can
                 // be requested using pagination.
-                if (client.isRoomEncrypted(roomId) && eventIndex !== null) {
+                if (this.state.isRoomEncrypted && eventIndex !== null) {
```

Comment:
Same kind of question here… will this method only get called after `this.state.isRoomEncrypted` is loaded? If not, is that okay?

Reply:
I conditioned the `TimelinePanel` under the `isRoomEncryptedLoaded` check. So this case should not happen now